pkg/trafficpolicy/trafficpolicy.go

```diff
@@ -56,3 +56,81 @@ func (out *OutboundTrafficPolicy) AddRoute(httpRouteMatch HTTPRouteMatch, weight
 	}
 	return nil
 }
+
+// MergeInboundPolicies merges latest InboundTrafficPolicies into original InboundTrafficPolicies
```

Comment:
The term `original` is a bit unclear to me. Maybe another line on what `original` refers to here would be helpful.

Reply:
Agreed